brands/shop/convenience.json

```diff
@@ -1291,6 +1280,15 @@
       "shop": "convenience"
     }
   },
+  "shop/convenience|Mesra": {
+    "locationSet": {"include": ["my"]},
+    "tags": {
+      "brand": "Petronas",
```

Comment:
Why does this `brand` and `brand:wikidata` point to "Petronas"? 
Are these convenience stores that always are at Petronas fuel stations?
It would be better if they had their own wikidata for the separate convenience store brand.